lib/standalone.js

```diff
@@ -118,7 +118,15 @@ module.exports = function(
       })
       .catch(handleError)
       .then(stylelintResult => {
-        return prepareReturnValue([stylelintResult]);
+        const returnValue = prepareReturnValue([stylelintResult]);
+        const postcssResult = stylelintResult._postcssResult;
+        // If we're fixing the output should be the fixed code
```

Comment:
`// If we're fixing, the output should be the fixed code` the added comma here helps readability